stubs/tensorflow/tensorflow/raw_ops.pyi

```diff
@@ -0,0 +1,45 @@
+from _typeshed import Incomplete
+from typing import Literal
+
+from tensorflow import Tensor
+from tensorflow._aliases import TensorCompatible
+
+def Fingerprint(data: TensorCompatible, method: Literal["farmhash64"], name: str | None = None) -> Tensor: ...
+def Snapshot(input: TensorCompatible, name: str | None = None) -> Tensor: ...
+def ResourceApplyAdagradV2(
+    var: Tensor,
+    accum: Tensor,
+    lr: TensorCompatible,
+    epsilon: TensorCompatible,
+    grad: TensorCompatible,
+    use_locking: bool = False,
+    update_slots: bool = True,
+    name: str | None = None,
+) -> None: ...
```

Comment:
This returns "The created Operation" according to its docstring.

Reply:
Fixed in b392c8a8e.